features/remotepollerng/daemon/src/main/java/org/opennms/netmgt/remotepollerng/RemotePollerd.java

```diff
@@ -358,4 +391,49 @@ private String getServiceParameter(final Service svc, final String key) {
     @Override
     public void afterPropertiesSet() throws Exception {
     }
+
+    public static class RemoteLatencyResource extends AbstractResource {
```

Comment:
Can we move this to the `org.opennms.netmgt.collection.support.builder` package of the `org.opennms.features.collection.api` module?

I've been trying to collect all of the resource types there :)

Reply:
Done